packages/components/psammead-image/src/testHelpers/fixtureData.js

```diff
@@ -0,0 +1,36 @@
+export const landscape = {
+  alt: 'Student sitting an exam',
+  src:
+    'https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg',
+  srcset:
+    'https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg 1024w, https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg 1024w, https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg 1024w, https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg 1024w, https://ichef.bbci.co.uk/news/1024/cpsprodpb/7098/production/_104842882_students.jpg 1024w',
```

Comment:
Are these supposed to have real values, or are we waiting on UX input? It took me an embarassing amount of time to clock why I couldn't affect the behaviour 😂

Reply:
Ha - yes these should be updated 👍 https://github.com/BBC-News/simorgh/issues/978 covers the image sizes but we can choose some arbitrary ones for now.

I'd also suggest refactoring like:

```js
const sizes = [300, 600, 1200];
const imageUrl = `https://ichef.bbci.co.uk/news/[WIDTH]/cpsprodpb/7098/production/_104842882_students.jpg`;
export const landscape = {
  alt: 'Student sitting an exam',
  src: imageUrl,
  srcset: sizes.map(size => imageUrl.replace('[WIDTH]', size)),
```